libsplinter/src/registry/mod.rs

```diff
@@ -28,9 +28,9 @@
 pub mod client;
 #[cfg(feature = "diesel")]
 mod diesel;
-mod error;
 #[cfg(feature = "rest-api-actix-web-1")]
 mod rest_api;
+pub mod error;
```

Comment:
I think we're tending towards re-exporting the items that should be public, vs making the module public

Reply:
Made module private, relevant types are re-exported bellow.